docs/multicluster/architecture.md

```diff
@@ -6,13 +6,15 @@ ClusterSet. Antrea Multi-cluster also supports Antrea ClusterNetworkPolicy repli
 Multi-cluster admins can define ClusterNetworkPolicies to be replicated across the entire
 ClusterSet, and enforced in all member clusters.
 
-The diagram below depicts a basic multi-cluster topology in Antrea.
+The diagram below depicts a basic Antrea Multi-cluster topology with one leader cluster
+and two member clusters.
 
-<img src="assets/basic-topology.png" width="500" alt="Antrea Multi-cluster Topology">
+<img src="assets/basic-topology.svg" width="650" alt="Antrea Multi-cluster Topology">
 
-Given a set of Kubernetes clusters, there will be a leader cluster and several member clusters.
-By default, a leader cluster itself is also a member cluster of a ClusterSet. A cluster
-can also be configured as a dedicated leader cluster of multiple ClusterSets.
+An Antrea Multi-cluster ClusterSet includes a leader cluster and multiple member clusters.
+Antrea Multi-cluster Controller needs to be deployed in the leader and all member
+clusters. A cluster can serve as the leader, and meanwhile also be a member cluster of the
+ClusterSet.
 
 ## Terminology
 
```

Comment:
Fix the following:

> member of a ClusterSet with unique cluster ID, to claim a ClusterSet with unique ClusterSet ID.

with a unique cluster ID

and to claim a ClusterSet with a unique ClusterSet ID.

Reply:
It might be more natural to talk about ClusterSet first:

The ClusterClaim CRD is used to claim a ClusterSet with a unique ClusterSet ID, and to claim the cluster itself as a member of a ClusterSet with a unique cluster ID.

Reply:
Done